robolectric/src/test/java/org/robolectric/shadows/ShadowPackageManagerTest.java

```diff
@@ -1278,14 +1163,14 @@ public void getPermissionInfo_noMetaData() throws Exception {
 
   @Test
   public void getPermissionInfo_withMetaData() throws Exception {
-    PermissionInfo permission = packageManager.getPermissionInfo("some_permission", PackageManager.GET_META_DATA);
+    PermissionInfo permission = packageManager.getPermissionInfo("org.robolectric.some_permission", PackageManager.GET_META_DATA);
```

Comment:
Let's summarize this and other behavior changes for release notes.

Reply:
Started a section for 3.5, we can move it depending what release this goes into.